packages/cli-debugger-ui/src/ui/index.html

```diff
@@ -23,16 +23,18 @@
         <p>
           Please use the
           <strong>Open Debugger</strong> workflow to debug apps using Hermes
```

Comment:
Note to self: link will be `http://reactnative.dev/docs/next/debugging#opening-the-debugger`.